ui/app/AppLayouts/Onboarding2/LoginWithKeycardFlow.qml

```diff
@@ -59,11 +60,13 @@ SQUtils.QObject {
         KeycardIntroPage {
             keycardState: root.keycardState
             displayPromoBanner: root.displayKeycardPromoBanner
-            unlockUsingSeedphrase: true
+            unblockUsingSeedphraseAvailable: true
+            unblockWithPukAvailable: root.remainingAttempts === 1 || root.remainingAttempts === 2
```

Comment:
The remaining attempts starts at 3. I just want to confirm that the purpose of the property is really to activate after the first failure?

Reply:
Yup, intended

Reply:
But to be confirmed with @benjthayer ; there's still some unclear corner case as to what to display when